packages/web-core/src/api/v2/base.ts

```diff
@@ -19,7 +19,7 @@ import type { Configuration } from './configuration';
 import type { AxiosPromise, AxiosInstance, AxiosRequestConfig } from 'axios';
 import globalAxios from 'axios';
 
-export const BASE_PATH = "https://<project ID>.frontendapi.corbado.io".replace(/\/+$/, "");
+export const BASE_PATH = "http://localhost".replace(/\/+$/, "");
```

Comment:
@incorbador  this was done by auto-generated code. Should I remove this?

Reply:
I think we can leave this as is because we always set the path explicitly and never rely on the default